tests/test_core.py

```diff
@@ -5966,9 +5966,9 @@ def run_all(x):
 
     run_all('lto')
 
-  def test_autodebug(self):
+  def test_autodebug_bitcode(self):
     if self.is_wasm_backend():
-      # autodebugging only works with bitcode objects
+      # bitcode autodebugging only works with bitcode objects
```

Comment:
Better to lift this out into a "@no_wasm_backend" perhaps?

Reply:
Sorry... maybe better to skip here `if get_setting('WASM_OBJECT_FILES') == 0` (which is true for the LTO mode of testing).

Reply:
Thanks, changed to that.